py/API_operations/Subset.py

```diff
@@ -163,15 +163,15 @@ def _db_fetch(self, object_ids: ObjectIDListT) -> Iterable[DBObjectTupleT]:
         )
         ret = (
             ret.outerjoin(Image, ObjectHeader.all_images)
-            .outerjoin(ObjectCNNFeature)
+            .outerjoin(ObjectCNNFeatureVector)
             .join(ObjectFields)
         )
         ret = ret.filter(ObjectHeader.objid == any_(object_ids))
         ret = ret.order_by(ObjectHeader.objid, Image.imgid)
         ret = ret.with_entities(
             ObjectHeader,
             ObjectFields,
-            ObjectCNNFeature,
+            ObjectCNNFeatureVector,
```

Comment:
J'ai un doute. Ca a été essayé?

Reply:
<img width="1310" alt="Capture d’écran 2024-02-17 à 13 46 38" src="https://github.com/ecotaxa/ecotaxa_back/assets/2922559/67b164f0-ddc5-4bce-9ab0-467b2bf6db38">

Il y a une seule fonction mentionnant CNNFeature à savoir add_cnn_features qui n'est pas appelé, mais add_cnn_features_with_pk l'est.

Toutes les autres lignes des fichiers avec CNNFeatures à savoir (lien avec le http.server qui tourne dans `lovnower@niko:~/git/ecotaxa_back/QA/py/htmlcov$ python -m http.server 9000`)

- http://localhost:9000/_home_lovnower_git_ecotaxa_back_py_API_operations_Subset_py.html#t164
   - pragma no cov error
- http://localhost:9000/_home_lovnower_git_ecotaxa_back_py_DB_CNNFeatureVector_py.html#t22
- http://localhost:9000/_home_lovnower_git_ecotaxa_back_py_BO_Prediction_py.html#t79
   - l 109 : pragma no cov scale
   - l 88 : si la DB prévoit plusieurs photo par object_id ce n'est pas le cas en réalité (cf discussion JO VR 22-23)
- http://localhost:9000/_home_lovnower_git_ecotaxa_back_py_API_operations_SimilaritySearch_py.html
   - l62,63 : feature_extractor_selected needs to be selected in project table (or not anymore with new interface since 12/23)
- http://localhost:9000/_home_lovnower_git_ecotaxa_back_py_DB_helpers_DBWriter_py.html#t40
   - 2 lines not called defining add_cnn_features, one line linked to class_histo
- http://localhost:9000/_home_lovnower_git_ecotaxa_back_py_DB_Relations_py.html#t130